mapreduce/src/main/java/com/marklogic/mapreduce/ContentWriter.java

```diff
@@ -777,7 +777,7 @@ public int getTransactionSize(Configuration conf) {
             int txnSize = conf.getInt(TXN_SIZE, 0);
             return txnSize <= 0 ? 1 : txnSize;
         } 
-        return 1000 / conf.getInt(BATCH_SIZE, DEFAULT_BATCH_SIZE);
+        return DEFAULT_TXN_SIZE;
```

Comment:
Why are we no longer calculating transaction size, but always return default transaction size?

Reply:
I'm changing the case where there is no transaction_size option passed through the command line, so the default case. I think the expected behavior is no matter what batch_size the user specifies, we always want the default transaction size to be 1. Is my understanding correct?